ground/src/main/java/com/google/android/ground/model/submission/MultipleChoiceResponse.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.ground.model.submission
+
+import com.google.android.ground.model.task.MultipleChoice
+import com.google.android.ground.model.task.Option
+import java8.util.Optional
+import kotlinx.serialization.Serializable
+
+/**
+ * User responses to a select-one (radio) or select-multiple (checkbox) field.
+ */
+@Serializable
+class MultipleChoiceResponse(
+    private val multipleChoice: MultipleChoice?, val selectedOptionIds: List<String>
```

Comment:
Why is `multipleChoice` nullable now? It wasn't in the original java class.

Reply:
I think I saw that there were callers that were passing null so I updated the field to be null to be more correct, but after looking again I realized that it was just in tests that null was passed. I've now updated the callers and made the field non nullable again.